cmd/metal-api/internal/datastore/rethinkdb_integration_test.go

```diff
@@ -5,43 +5,61 @@ package datastore
 
 import (
 	"context"
+	"os"
 
+	"github.com/google/go-cmp/cmp"
+	"github.com/google/go-cmp/cmp/cmpopts"
 	"github.com/metal-stack/metal-api/cmd/metal-api/internal/metal"
 	"github.com/metal-stack/metal-api/test"
-	"go.uber.org/zap/zaptest"
+	"github.com/testcontainers/testcontainers-go"
+	"go.uber.org/zap"
 
 	"testing"
-
-	"github.com/stretchr/testify/require"
 )
 
-func TestRethinkStore_ConflictIsReturned(t *testing.T) {
```

Comment:
This is now already covered in a switch test and will also be covered in all other tests if we start to test like this for the rest of the package. Hence removed.